website/src/components/LanguageSelector/LanguageSelector.tsx

```diff
@@ -37,7 +37,8 @@ const LanguageSelector = () => {
       const locale = option.target.value;
       setCookie("NEXT_LOCALE", locale, { path: "/" });
       const path = router.asPath;
-      return router.push(path, path, { locale });
+      await router.push(path, path, { locale });
+      router.reload();
```

Comment:
Although it's obvious that this will work is there a better way that we can use `mutate` to be re-validate the task data?

Reply:
Right now I want to make it work correctly.  I will leave it to others to find a more elegant solution.